projects/views.py

```diff
@@ -0,0 +1,22 @@
+from django.db.models import Q
+from dal import autocomplete
+
+from .models import Client
+
+
+class ClientAutocomplete(autocomplete.Select2QuerySetView):
+    def get_queryset(self):
+        user = self.request.user
+
+        if not (user.is_authenticated() and user.is_staff):
+            return Client.objects.none()
```

Comment:
In the future, we can make this endpoint be public if we have a public form that uses it.

However, since the only form that currently uses it is in the admin UI, I figure we should have the endpoint have the same security constraints as the admin UI for now.